code/controllers/subsystem/dynamic/dynamic.dm

```diff
@@ -387,10 +387,11 @@ SUBSYSTEM_DEF(dynamic)
 		return advisory_string
 
 	switch(round(shown_threat))
-		if(0)
+		//a white dwarf shift leads to a green security alert on report and special announcement, this prevents a meta check if the alert report is fake or not.
+		if(0 && round(threat_level) == 0)
```

Comment:
This isn't how switches work, it needs to be moved out of the switch

Reply:
done